BuildResidentialHPXML/measure.rb

```diff
@@ -3792,7 +3792,7 @@ def argument_errors(args)
     error = (args[:geometry_unit_num_floors_above_grade] == 1 && args[:geometry_attic_type] == HPXML::AtticTypeConditioned)
     errors << 'Units with a conditioned attic must have at least two above-grade floors.' if error
 
-    error = ((args[:water_heater_type] == HPXML::WaterHeaterTypeCombiStorage) || (args[:water_heater_type] == HPXML::WaterHeaterTypeCombiTankless)) && (args[:heating_system_type] != HPXML::HVACTypeBoiler)
+    error = [HPXML::WaterHeaterTypeCombiStorage, HPXML::WaterHeaterTypeCombiTankless].include?(args[:water_heater_type]) && !args[:heating_system_type].include?(HPXML::HVACTypeBoiler)
```

Comment:
When we introduced shared boilers in the measure, all the code in the measure was changed to `args[:heating_system_type].include?(HPXML::HVACTypeBoiler)`, but this method appears to have been missed.